recipes/cmake/package.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+@early()
```

Comment:
overkill no? `name = "cmake"` is a lot smaller..

Reply:
reduced